index.html

```diff
@@ -929,7 +929,7 @@ <h2>Fragment</h2>
 
         <pre class="example nohighlight"
              title="A resource external to a DID Document">
-did:example:123?service=agent&amp;relativeRef=/credentials#degree
+did:example:123?service=agent&amp;relativeRef=%2Fcredentials%23degree
```

Comment:
Don't need to encode `/` in the query segment. 

Do need to encode `#` within the query segment (i.e., when it's not being the fragment delimiter).
```suggestion
did:example:123?service=agent&amp;relativeRef=/credentials%23degree
```